code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -1948,16 +1948,29 @@ datum
 
 
 				else if (method == TOUCH)
-					if(iscarbon(M))
-						if(!M.wear_mask)
-							M.reagents.add_reagent("capsaicin",round(volume_passed/5))
-							if(prob(50))
-								M.emote("scream")
-								boutput(M, "<span class='alert'><b>Your eyes hurt!</b></span>")
-								M.take_eye_damage(1, 1)
-							M.change_eye_blurry(3)
-							M.changeStatus("stunned", 2 SECONDS)
-							M.change_misstep_chance(10)
+					if(ishuman(M))
+						var/mob/living/carbon/human/H = M
+						if((H.wear_mask?.c_flags & COVERSEYES) || (H.head?.c_flags & COVERSEYES) || H.glasses && istype(H.glasses, /obj/item/clothing/glasses/sunglasses/sechud))
```

Comment:
Why sechud

Reply:
I added sechuds as protection due to sec having access to grenades that puts out a cloud of capsaicin, (AKA, the crowd dispersal grenades.)  and HoS's hat didn't provide protection from the gas.

Reply:
Why do they not cover eyes

Reply:
If your talking about the sechuds, currently all glasses count as COVERSEYES, even if some of them don't make sense. I didn't want to have all glasses provide protection against it, however I can change it to use the COVERSEYES proc for glasses if necessary. (Like prescription glasses and eyepatches) Although, a pr to change which glasses has the COVERSEYES proc would help with this. If your talking about the HoS hat, apparently that was changed some time ago to no longer have the COVERSEYES proc.